src/main/java/com/amazonaws/services/sqs/AmazonSQSTemporaryQueuesClientBuilder.java

```diff
@@ -5,6 +5,8 @@
 
 public class AmazonSQSTemporaryQueuesClientBuilder {
 
+    public final static long QUEUE_RETENTION_PERIOD_SECONDS_DEFAULT = 300;
```

Comment:
```suggestion
    public final static long IDLE_QUEUE_RETENTION_PERIOD_SECONDS_DEFAULT = 300;
```
(and the references too of course :))